tools/llvm-cda-dump.cpp

```diff
@@ -389,8 +389,24 @@ int main(int argc, char *argv[]) {
                      << options.dgOptions.entryFunction << "\n";
         return 1;
     }
+    std::unique_ptr<LLVMPointerAnalysis> pta{nullptr};
+    if (use_pta) {
+        auto& ptaopts = options.dgOptions.PTAOptions;
+#ifdef HAVE_SVF
+        if (ptaopts.isSVF()) {
+            pta.reset(new SVFPointerAnalysis(M.get(), ptaopts));
+            pta->run();
+        } else
+#endif // HAVE_SVF
```

Comment:
This change causes the link failures in CI. `llvm-cda-dump` should be linked with SVF.